src/config/Config.ts

```diff
@@ -24,6 +24,9 @@ const integrationTypes: Record<IntegrationType, IntegrationTypeConfigBase> = {
     [IntegrationType.SPLUNK]: {
         name: 'Splunk'
     },
+    [IntegrationType.SERVICE_NOW]: {
+        name: 'Service now'
```

Comment:
`s/Service now/ServiceNow/`

Reply:
Whoops - didn't know how the name of the service was spelt, thanks!